diff --git a/sound/usb/quirks.c b/sound/usb/quirks.c
index ab9f3da49941..a09d693f7ff2 100644
--- a/sound/usb/quirks.c
+++ b/sound/usb/quirks.c
@@ -1922,6 +1922,8 @@ static const struct usb_audio_quirk_flags_table quirk_flags_table[] = {
 		   QUIRK_FLAG_DSD_RAW),
 	VENDOR_FLG(0x22d9, /* Oppo */
 		   QUIRK_FLAG_DSD_RAW),
+	VENDOR_FLG(0x22e8, /* Cambridge */
+		   QUIRK_FLAG_DSD_RAW),
 	VENDOR_FLG(0x23ba, /* Playback Design */
 		   QUIRK_FLAG_CTL_MSG_DELAY | QUIRK_FLAG_IFACE_DELAY |
 		   QUIRK_FLAG_DSD_RAW),
